### PR TITLE
Add metrics for SQS

### PR DESCRIPTION
**Pull request by hunterwerlla**
## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: telemetry/definitions/commonDefinitions.json [hunk 1]
```diff
@@ -177,6 +178,11 @@
             "type": "string",
             "allowedValues": ["IAM", "SecretsManager"],
             "description": "How the database credentials are being retrieved"
+        },
+        {
+            "name": "sqsQueueType",
+            "allowedValues": ["standard", "fifo"],
+            "description": "The type of an SQS Queue (either standard or fifo)"
```
Comment (justinmk3) [inline telemetry/definitions/commonDefinitions.json:185]: nit: if we have valid values enumerated programmatically, better to leave them out of descriptions (or auto-generate the description). Small thing, but adds up to a maintenance cost over time

File: telemetry/definitions/commonDefinitions.json [hunk 2]
```diff
@@ -537,6 +543,57 @@
...
+        {
+            "name": "sqs_deleteMessages",
+            "description": "Delete one or more messages from an SQS queue. Value indicates the number of messages that we tried to delete.",
+            "unit": "Count",
+            "metadata": [{ "type": "result" }, { "type": "sqsQueueType" }]
+        },
+        {
+            "name": "sqs_copyMessage",
+            "description": "Copy a message to the clipboard",
+            "metadata": [{ "type": "sqsQueueType" }]
```
Comment (justinmk3) [inline telemetry/definitions/commonDefinitions.json:576]: we might instead want a generic `copyToClipboard` metric with perhaps a parameter that indicates the component that initiated it. 

Though having telemetry on copy/paste in general seems a bit much, almost like tracking how many keys the user pressed...

**Review by justinmk3 — APPROVED** [review]

File: telemetry/definitions/commonDefinitions.json (hunk 2 above)
Comment (hunterwerlla) [reply telemetry/definitions/commonDefinitions.json:576]: Yeah, maybe we should just not have this. It doesn't really tell us anything, removed.

File: telemetry/definitions/commonDefinitions.json (hunk 1 above)
Comment (hunterwerlla) [reply telemetry/definitions/commonDefinitions.json:185]: Sure, removed from the description

**Pull request merged**